src/main/java/allez/Allez.java

```diff
@@ -0,0 +1,206 @@
+package allez;
+
+import allez.task.Deadline;
+import allez.task.Event;
+import allez.task.Task;
+import allez.task.ToDo;
+
+import java.util.Scanner;
+
+
+public class Allez {
+    public static final int LIST_SIZE = 100;
+    protected static Task[] tasks = new Task[LIST_SIZE];
+    protected static  int taskCount = 0;
+    protected static Scanner in = new Scanner(System.in);
+
+    public enum TaskType {
+        TODO, DEADLINE, EVENT
+    }
+
+    public static void printList(Task[] tasks){
+        int count = 0;
+
+        if(tasks[0] == null) {
+            System.out.println("List is currently empty");
+            return;
+        }
+
+        System.out.println("Here are the tasks in your list:");
+        while(tasks[count] != null){
+            System.out.println("\t" + (count+1) + ". " + tasks[count].toString());
+            count+=1;
+        }
+    }
+
+    public static void main(String[] args) {
+        boolean hasEnded = false;
+
+        printGreeting();
+
+        while(!hasEnded) {
+            hasEnded = executeCommands();
+        }
+
+        printExit();
+    }
+
+    private static boolean executeCommands() {
+        String line = in.nextLine();
+        String firstWord = line.split(" ",2)[0];
+
+        switch(firstWord){
```

Comment:
There needs to be a gap between switch and ( , and,  ) and {, as per coding standards